github-actions/mima.sbt

```diff
@@ -1,23 +1,17 @@
 import com.typesafe.tools.mima.core._
 mimaBinaryIssueFilters ++= Seq(
-  "org.typelevel.sbt.gha.WorkflowJob.withId",
-  "org.typelevel.sbt.gha.WorkflowJob.withName",
-  "org.typelevel.sbt.gha.WorkflowJob.withSteps",
-  "org.typelevel.sbt.gha.WorkflowJob.withSbtStepPreamble",
-  "org.typelevel.sbt.gha.WorkflowJob.withCond",
-  "org.typelevel.sbt.gha.WorkflowJob.withPermissions",
-  "org.typelevel.sbt.gha.WorkflowJob.withEnv",
-  "org.typelevel.sbt.gha.WorkflowJob.withOses",
-  "org.typelevel.sbt.gha.WorkflowJob.withScalas",
-  "org.typelevel.sbt.gha.WorkflowJob.withJavas",
-  "org.typelevel.sbt.gha.WorkflowJob.withNeeds",
-  "org.typelevel.sbt.gha.WorkflowJob.withMatrixFailFast",
-  "org.typelevel.sbt.gha.WorkflowJob.withMatrixAdds",
-  "org.typelevel.sbt.gha.WorkflowJob.withMatrixIncs",
-  "org.typelevel.sbt.gha.WorkflowJob.withMatrixExcs",
-  "org.typelevel.sbt.gha.WorkflowJob.withRunsOnExtraLabels",
-  "org.typelevel.sbt.gha.WorkflowJob.withContainer",
-  "org.typelevel.sbt.gha.WorkflowJob.withEnvironment",
-  "org.typelevel.sbt.gha.WorkflowJob.withConcurrency",
-  "org.typelevel.sbt.gha.WorkflowJob.withTimeoutMinutes"
+  "org.typelevel.sbt.gha.WorkflowJob.with*",
+  "org.typelevel.sbt.gha.WorkflowJob.append*",
+  "org.typelevel.sbt.gha.WorkflowJob.update*",
+  "org.typelevel.sbt.gha.WorkflowStep.update*",
+  "org.typelevel.sbt.gha.WorkflowStep.add*",
+  "org.typelevel.sbt.gha.WorkflowStep#Sbt.with*",
+  "org.typelevel.sbt.gha.WorkflowStep#Sbt.add*",
+  "org.typelevel.sbt.gha.WorkflowStep#Sbt.update*",
+  "org.typelevel.sbt.gha.WorkflowStep#Run.with*",
+  "org.typelevel.sbt.gha.WorkflowStep#Run.add*",
+  "org.typelevel.sbt.gha.WorkflowStep#Run.update*",
+  "org.typelevel.sbt.gha.WorkflowStep#Use.with*",
+  "org.typelevel.sbt.gha.WorkflowStep#Use.add*",
+  "org.typelevel.sbt.gha.WorkflowStep#Use.update*"
```

Comment:
This scares me a little bit vs. explicit enumeration of exceptions, but the pattern is obvious enough I won't stand in the way.

Reply:
Happy to enumerate them if you prefer.  I just used this because @armanbilge mentioned it last time I added to this.  https://github.com/typelevel/sbt-typelevel/pull/728#discussion_r1702855254

Reply:
I defer.

Reply:
Because these classes are `sealed` actually any `ReversedMissingMethodProblem` can be ignored.